packages/ui-components/src/components/calendar/calendar.tsx

```diff
@@ -39,18 +36,28 @@ export class KvCalendar implements ICalendar, ICalendarEvents {
 	/** @inheritdoc */
 	@Prop({ reflect: false }) initialDate?: string;
 	/** @inheritdoc */
+	@Prop({ reflect: false }) hoveredDate?: string;
+	/** @inheritdoc */
 	@Prop({ reflect: false }) disabledDates?: string[] = [];
 	/** @inheritdoc */
 	@Prop({ reflect: false }) minDate?: string;
 	/** @inheritdoc */
 	@Prop({ reflect: false }) maxDate?: string;
+	/** @inheritdoc */
+	@Prop({ reflect: false }) displayPreviousMonthArrow?: boolean = true;
```

Comment:
`canNavigateToPreviousMonth`

Reply:
I don't agree with that name here. This prop is only used to control the presence of the arrow on the left or on the right. The navigation can be enabled or not if the minimum or maximum date has been reached. Therefore, the name can be a little bit deceiving.